e2e/cypress/integration/account_settings/display/account_settings_position_spec.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @account_setting
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Account Settings > General > Position', () => {
+    let testTeam;
+    let testUser;
+    const position = 'Master hacker';
+
+    beforeEach(() => {
+        cy.apiAdminLogin();
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+
+            cy.apiLogin(testUser);
+
+            // # Visit town square
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-T2063 Position', () => {
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on Account settings menu item
+        cy.findByText('Account Settings').should('be.visible').click();
+
+        // # Fill-in the position field
+        cy.findByText('Position').should('be.visible').click();
+        cy.get('#position').type(position);
+        cy.get('#saveSetting').click();
+
+        // # Exit the modal
+        cy.get('body').type('{esc}', {force: true});
+
+        // # Post message in the main channel
+        cy.postMessage('hello from master hacker');
+
+        // # Click on the profile image
+        cy.get('.profile-icon > img').as('profileIconForPopover').click();
+
+        // # Verify that the popover is visible
+        cy.get('#user-profile-popover').should('be.visible').within(() => {
```

Comment:
it might be easier to use contains
```suggestion
        cy.contains('#user-profile-popover', 'Master hacker').should('be.visible');
```
although I'm suggesting this from memory, and it might fail